pkg/cmd/bootstrap.go

```diff
@@ -108,9 +110,10 @@ func (io *BootstrapParameters) Complete(name string, cmd *cobra.Command, args []
 		return err
 	}
 
-	if cmd.Flags().NFlag() == 0 {
-		return initiateInteractiveMode(io, client)
+	if cmd.Flags().NFlag() == 0 || cmd.Flag("interactive").Changed {
```

Comment:
You could simplify this to just...

```
if cmd.Flags().NFlag() == 0 || io.Interactive {
}
```

Reply:
Thanks. I'll update all similar references to this form.